main.go

```diff
@@ -167,7 +178,7 @@ func ProcessKeptnCloudEvent(ctx context.Context, event cloudevents.Event) error
  * Environment Variables
  * env=runlocal   -> will fetch resources from local drive instead of configuration service
  */
-var cloudEventListener = CloudEventListener
+// var cloudEventListener = CloudEventListener
```

Comment:
We might need it again because we want to improve the test where it was used. For now we have just commented the test and the cloudEventListener variable.